scripts/station_csv_to_json.py

```diff
@@ -33,12 +48,19 @@
                     row[STATION_CODE], row[STATION_NAME]))
                 continue
             STATION_COUNT = STATION_COUNT + 1
+            ecodivision_name = fetch_ecodivision_name(row[STATION_LATITUDE], row[STATION_LONGITUDE], ECODIVISIONS)
+            if ecodivision_name is not None:
+                core_season = CORE_SEASONS[ecodivision_name]['core_season']
+            else:
+                core_season = None #pylint: disable=invalid-name
```

Comment:
It's kinda funny that we have to disable pylint for this. Is there some kind of conventions when using uppercase for variables in Python?

Reply:
I was trying to conform to UPPERCASE for constants - but I'm ready to through that out completely. Python doesn't have constants - so I'm ready to just change that in our linting settings and forget about uppercase variables.